arangod/Scheduler/SchedulerFeature.cpp

```diff
@@ -61,8 +61,8 @@ size_t defaultNumberOfThreads() {
   // use two times the number of hardware threads as the default
   size_t result = arangodb::NumberOfCores::getValue() * 2;
   // but only if higher than 64. otherwise use a default minimum value of 64
```

Comment:
Comment is outdated now
```suggestion
  // but only if higher than 64. otherwise use a default minimum value of 32
```